controller/bin/boot

```diff
@@ -10,14 +10,21 @@ set -eo pipefail
 [[ $DEBUG ]] && set -x
 
 # configure etcd
-export ETCD_PORT=${ETCD_PORT:-4001}
-export ETCD="$HOST:$ETCD_PORT"
+# if ETCDCTL_PEERS isn't already set, set it
+if [ -z "$ETCDCTL_PEERS" ]; then
+	export ETCDCTL_PEERS="http://${ETCD_HOST:-${HOST:-127.0.0.1}}:${ETCD_PORT:-4001}"
+fi
+
+# ETCD is used by settings.py
+IFS=',' read -ra ETCD_PEERS_ARR <<< "$ETCDCTL_PEERS"
+export ETCD=$(echo ${ETCD_PEERS_ARR[0]} | awk -F/ '{print $3}')
```

Comment:
We don't use this variable in the script any more, IIRC. Let's remove this confusing array parse logic if we're not using it.